### ISSUE TITLE
Pre-trained model sharing?

### ISSUE THREAD

**Issue by dreamibor** [Kyubyong/tacotron#107]
Hi, Thank you for sharing your code, I am currently looking for a pre-trained tacotron model, it seems you have talked about this in previous issues, but I cannot find any links to pre-trained models. Did you remove that? Probably there are some legal issues? Thank you.

**Comment by Kyubyong** [comment]
Oh my god! Absolutely no. There's no legal issues at all when I share my pretrained model on the LJ dataset or the WEB data. I was totally forgetting it. And what is even worse is I can't find them! Let me retrain them. It should take around five days. Please check them again in five days. Sorry.

**Comment by dreamibor** [comment]
@Kyubyong That will be much appreciated! Thank you.

**Comment by Kyubyong** [comment]
I've just updated README.md with the links for pretrained models. I'm not lazy so I didn't check if they work, so please let me know if there's some problem with them.

**Comment by dreamibor** [comment]
@Kyubyong I tried with your newly uploaded checkpoint, and it looks good. The quality of generated voice is more than acceptable. Thank you.

**Comment by dreamibor** [comment]
Closed for the author has uploaded the model checkpoint.

**Comment by The-Gupta** [comment]
@Kyubyong ,This is not an issue, still...
Could you tell a measure of accuracy for all the pretrained models with respect to the state-of-the-art TTS service available?
